### PR TITLE
Move solution file to the repo root

### PR DESCRIPTION
**Pull request by tmat**
To align with other repos.

### CODE REVIEW

**Comment by 333fred** [comment]
Analyzer tutorial: https://github.com/dotnet/roslyn-analyzers/blob/569bd373a4831d3035597197e02980b57602a7f2/src/MetaCompilation.Analyzers/Core/ReadMe.md

**Comment by tmat** [comment]
@333fred Done. The tutorial readme.md seems to be a bit buried. Any reason why it's not in the docs directory?

**Comment by tmat** [comment]
@mavasani  The tutorial says VS 2015 is required. As we moved to dotnet SDK VS 2017 is required, right?

File: README.md [hunk 1]
```diff
@@ -101,17 +101,6 @@ Getting Started
 
 Execute `cibuild.cmd` to clean, restore, build and runs tests
 
-**NOTE** The current build of System.Reflection.Metadata (from NuGet package System.Reflection.Metadata.1.2.0-rc2-23629) is unsigned. This causes unit tests to fail. To work around this problem until a new, signed version is available, give the command
```
Comment (333fred) [inline README.md]: Can't comment on the specific line, but the getting started instructions above this need to be updated.

**Comment by mavasani** [comment]
@tmat yes we need to fix documentation to point to VS2017

**Comment by mavasani** [comment]
I think the tutorial itself is still not migrated and not part of the solution.

**Comment by tmat** [comment]
OK, will do that later.

**Comment by tmat** [comment]
@333fred Looks good now?

**Pull request merged**